src/utils.js

```diff
@@ -75,11 +107,11 @@ export function getElementSize(node) {
   }
 }
 
-export function getMeasurements(node) {
+export function getMeasurements(node: HTMLElement) {
   const rect = node.getBoundingClientRect()
   return {
-    x: rect.x,
-    y: rect.y,
+    x: rect.left,
```

Comment:
Any particular reason not use `x` and `y` here?

Reply:
Flow does not describe this two. Seems like this type is IE/EDGE specific.
https://github.com/facebook/flow/blob/master/lib/dom.js#L3045-L3052

Reply:
Ah I see! Thanks for letting me know 😁